fbw-a32nx/src/systems/fmgc/src/guidance/vnav/descent/DescentStrategy.ts

```diff
@@ -1,18 +1,15 @@
-// Copyright (c) 2021-2023 FlyByWire Simulations
-//
-// SPDX-License-Identifier: GPL-3.0
-
+import { AircraftConfig } from '@fmgc/flightplanning/new/AircraftConfigInterface';
 import { AtmosphericConditions } from '@fmgc/guidance/vnav/AtmosphericConditions';
 import { FlightPathAngleStrategy, VerticalSpeedStrategy } from '@fmgc/guidance/vnav/climb/ClimbStrategy';
 import { FlapConf } from '@fmgc/guidance/vnav/common';
-import { AircraftConfiguration } from '@fmgc/guidance/vnav/descent/ApproachPathBuilder';
+import { AircraftConfiguration as AircraftCtlSurfcConfiguration } from '@fmgc/guidance/vnav/descent/ApproachPathBuilder';
 import { EngineModel } from '@fmgc/guidance/vnav/EngineModel';
 import { Predictions, StepResults } from '@fmgc/guidance/vnav/Predictions';
 import { VerticalProfileComputationParametersObserver } from '@fmgc/guidance/vnav/VerticalProfileComputationParameters';
 import { VnavConfig } from '@fmgc/guidance/vnav/VnavConfig';
 import { WindComponent } from '@fmgc/guidance/vnav/wind';
 
-export const DEFAULT_AIRCRAFT_CONFIG: AircraftConfiguration = {
+export const DEFAULT_AIRCRAFT_CTL_SURFC_CONFIG: AircraftCtlSurfcConfiguration = {
```

Comment:
Do not use contractions: `DEFAULT_AIRCRAFT_CONTROL_SURFACE_CONFIG`

Reply:
done